tech/languages/csharp/csharp-ide.md

```diff
@@ -6,45 +6,36 @@ order: 2
+| [JetBrains Rider](http://jetbrains.com/rider) | [Proprietary](https://www.jetbrains.com/store/license.html), free for [Education and OpenSource](https://www.jetbrains.com/store/#edition=discounts) | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |  | tarball |
+| [Visual Studio Code](https://code.visualstudio.com) with [C# plugin](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp) | Binary is [Proprietary](https://code.visualstudio.com/License/) Source Code is [MIT](https://github.com/Microsoft/vscode/blob/master/LICENSE.txt) | :heavy_check_mark: |  | :heavy_check_mark: |  | [flatpak](https://flathub.org/repo/appstream/com.visualstudio.code.flatpakref), copr & rpm |
 
-_.NET Core and Mono_
+## Installing Eclipse
 
-JetBrains Rider is full featured IDE for all things C#, and even F# on .NET Core. If you are familiar with other JetBrains products, such as the IntelliJ or Resharper, you might like this one as well.
-
-[jetbrains.com/rider](http://jetbrains.com/rider)
-
-## Visual Studio Code
-
-_.NET Core Only_
-
-Visual Studio Code (aka VS Code) is plugin-based IDE that by itself won't just run C#. You have to install the C# plugin, and NuGet Package Manager.
-
-Download the Visual Studio Code at [code.visualstudio.com](https://code.visualstudio.com) and the C# Extension at [marketplace.visualstudio.com](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp)
-
-## Eclipse
-
-_.NET Core and Mono_
+First install Eclipse and Eclipse Marketplace
+```
+$ sudo dnf install eclipse eclipse-mpc
+```
+then install the aCute extension from the [Marketplace](https://marketplace.eclipse.org/content/acute-c-edition-eclipse-ide-experimental)
 
-You can use the [aCute plugin](https://marketplace.eclipse.org/content/acute-c-edition-eclipse-ide-experimental) on top of the established and popular [Eclipse IDE](/tools/eclipse/about.html). This C# extension for Eclipse IDE relies on [OmniSharp](http://www.omnisharp.net) and [Language Server Protocol](https://github.com/Microsoft/language-server-protocol). It comes with complete edition features for C# in the Eclipse IDE, relying on the Eclipse IDE provides all typical features of a good IDE (Polyglot development, SCM, deployment...) and give access to a rich ecosystem of extensions for about all existing technologies.
+## Installing MonoDevelop
 
 ```
-$ sudo dnf install eclipse eclipse-mpc
+$ sudo dnf install monodevelop
 ```
-then install the aCute extension from [Eclipse Marketplace](https://marketplace.eclipse.org/content/acute-c-edition-eclipse-ide-experimental)
 
-## MonoDevelop
+## Installing JetBrains Rider
 
-_Mono only_
+Apply for Open Source [license](https://www.jetbrains.com/store/?fromMenu#edition=discounts) and download Rider at [jetbrains.com/rider](http://jetbrains.com/rider) or download the [Early Access](https://www.jetbrains.com/rider/eap) version.
```

Comment:
I believe the correct wording is 'license For Open Source projects'. Also please split those next two downloads into two lines (bullet points?) to properly distinguish two different versions
 - btw. isn't one of them like, better?
For beginner developers, I'd say stability /working with Fedora/ is a priority on this website.